### DISCUSSION TITLE
With the latest Cobbler, does the new UI work with it yet?

### DISCUSSION THREAD

**Discussion by chris1984** [cobbler/cobbler#3560] [Q&A] [answered]
I was hoping to upgrade from 2.x to latest but want to make sure the UI can be installed on the same machine and used to manage it like back in 2.x

**Comment by SchoolGuy** [comment, marked as answer]
TLDR: No.

Long version: Sadly the new UI is not complete yet. The Django-based approach came to its limits soon after the Python 3 upgrade (3.0.0). As such the community needs more people working on it so it can be used in production. The initial work is done since we already have enabled authentication and some simple tasks are working.

**Reply by chris1984** [reply]
Thanks @SchoolGuy I am trying to learn Angular, so maybe I can help out. Do you have any easy issues that are still available?

**Reply by SchoolGuy** [reply]
@chris1984 for sure. I will take a look at work and create more granular tasks. I will ping you in the UI repo once they are available.
